net_test.go

```diff
@@ -507,15 +555,13 @@ func TestTCPPushPull(t *testing.T) {
 
 	// Send our node state
 	header := pushPullHeader{Nodes: 3}
-	hd := codec.MsgpackHandle{
-		BasicHandle: codec.BasicHandle{
-			TimeNotBuiltin: !m.config.MsgpackUseNewTimeFormat,
-		},
-	}
+	hd := codec.MsgpackHandle{}
+	hd.TimeNotBuiltin = !m.config.MsgpackUseNewTimeFormat
+
```

Comment:
Callout: Similar to [here](https://github.com/hashicorp/memberlist/pull/325#discussion_r2161342333).